src/content/docs/en/welcome.mdx

```diff
@@ -5,30 +5,32 @@ description: "This wiki is meant to be the extended documentation for both end u
 
 Welcome to the Ultramarine Wiki! This wiki is designed to help users get settled into Ultramarine Linux.
 
-Developer and Package Maintainer Documentation has moved to [Fyra Developer](https://developer.fyralabs.com). If you want to get started with contributing, head to the [Contributing Section](/en/contributing/contributorguide).
+Developer and Package Maintainer Documentation has moved to [Fyra Developer](https://developer.fyralabs.com).
+
+If you want to get started with contributing, head to the [Contributing Section](/en/contributing/contributorguide).
 
 On this page, you can read about the project, so if you want to become familiar with important concepts and our philosophy, read this first!
 
 If you want to install Ultramarine Linux, head to the [Getting Ultramarine](/en/installation/getting) section.
 
 ## General Philosophy
 
-Ultramarine Linux is a spiritual successor to Korora Linux, intending to make an operating system that "just works" for people of all levels of experience.

```

Comment:
Does this not stand anymore?

Reply:
it does but we are moving away from the just works thing and the Korora Linux thing is less important now, it's been nearly 7 years since Korora ended. Ultramarine has evolved since it started, and should be able to stand for itself now

Reply:
You are absolutely right.